pages/api/harness.js

```diff
@@ -1,38 +1,58 @@
-import { Web3Storage, File } from 'web3.storage'
-
-const getAccessToken = () => {
-  return process.env.WEB3STORAGE_TOKEN
-}
-
-const makeStorageClient = () => {
-  return new Web3Storage({ token: getAccessToken() })
-}
-
-const makeFileObjects = (cid, data) => {
-  const buffer = Buffer.from(JSON.stringify(data))
-
-  const files = [new File([buffer], 'test.json')]
-
-  return files
-}
-
-const storeFiles = async files => {
-  console.log(files) // eslint-disable-line no-console
-  const client = makeStorageClient()
-  const cid = await client.put('test')
-  return cid
-}
-
-const handler = (req, res) => {
+import { makeFileObjects, storeFiles } from '../../util/web3Storage'
+import { getCommunities } from '../../util/fauna'
+import { getSourcecredContributions } from '../../util/sourcecred'
+
+export const aggregateThirdPartyData = async (
+  communityData,
+  sourceCredContributions,
+  coordinapeEpoch,
+) =>
+  communityData.members
+    .map(member => ({
+      ...member,
+      sourcredData: sourceCredContributions.filter(
+        contribution => contribution.name === member.username,
+      ),
+      coordinapeData: [
+        ...(member.coordinapeData ? member.coordinapeData : []),
+        coordinapeEpoch.find(
+          newEpoch =>
+            newEpoch.address &&
+            newEpoch.address.toLowerCase() === member.address.toLowerCase(),
+        ),
+      ],
+    }))
+    .flat()
```

Comment:
Feels like this should be separate for each integration, I don't think it will be scalable if every update from a 3rd party forces a remapping of every other integration.